File: 0961-n-repeated-element-in-size-2n-array/0961-n-repeated-element-in-size-2n-array.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& nums) {
        unordered_set<int>st;
        int ret=0;
        for(auto &i:nums){
            if(st.count(i)){
                ret=i;
                break;
            }
            else st.insert(i);
        }
        return ret;
    }
};

File: 0961-n-repeated-element-in-size-2n-array/0961-n-repeated-element-in-size-2n-array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
       Set<Integer> st = new HashSet<>();
       int ret=0;
       for(int n : nums){
            if(st.contains(n)){
                ret=n;
                break;
            }
            else st.add(n);
       } 
       return ret;
    }
}